src/sdc11073/wsdiscovery/networkingthread.py

```diff
@@ -154,18 +154,19 @@ def add_multicast_message(self,
     def _repeated_enqueue_msg(self,
                               msg: OutgoingMessage,
                               delay_params: _UdpRepeatParams):
-        initial_delay_ms = random.randint(0, delay_params.max_initial_delay_ms)
-        next_send = time.time() + initial_delay_ms / 1000.0
-        delta_t = random.randrange(delay_params.min_delay_ms, delay_params.max_delay_ms) / 1000.0  # millisec -> seconds
-        self._send_queue.put(self._EnqueuedMessage(next_send, msg, 1))
-        for i in range(delay_params.repeat):
-            next_send += delta_t
-            self._send_queue.put(self._EnqueuedMessage(next_send, msg, i + 2))
-            delta_t = min(delta_t * 2, delay_params.upper_delay_ms)
+        if not self._quit_send_event.is_set():

```

Comment:
please add log warning message when message is dropped. otherwise messages are silently dropped

Reply:
done